src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -69,6 +69,11 @@
         }
     };
 
+    private static final ImmutableMap<String, String> SPECIAL_CASE_FIELDS =
+    ImmutableMap.of ("permissions_update_interval", "permissions_update_interval_in_ms",
+                     "credentials_update_interval", "credentials_update_interval_in_ms",
+                     "roles_update_interval", "roles_update_interval_in_ms");
+
```

Comment:
plz remove, I handle in CASSANDRA-17166, old names and new names are both present with correct values

Reply:
Sure, done, thanks.